src/components/gltf-model-plus.js

```diff
@@ -274,7 +369,13 @@ AFRAME.registerComponent("gltf-model-plus", {
       }
 
       const gltfPath = THREE.LoaderUtils.extractUrlBase(src);
-      const model = await cachedLoadGLTF(src, this.data.basePath, this.data.contentType, this.preferredTechnique);
+
+      if (!GLTFCache[src]) {
+        GLTFCache[src] = loadGLTF(src, this.preferredTechnique);
+      }
+
+      const cachedModel = await GLTFCache[src];
```

Comment:
I wold inline this and get rid of `cachedModel` since its not really valid to use if for anything but cloning. But that's mostly personal preference.